### ISSUE TITLE
Station information not displaying

### ISSUE THREAD

**Issue by snvgglebear** [ts2/ts2-sim-server#35]
When I select a station the station information box does not display any information other than the station name. in the previous version it would display all scheduled service at that station.

**Comment by npiganeau** [comment]
Indeed. Actually, if you try right at the beginning of the simulation, it works correctly, but after a time (or an action?), it doesn't anymore. I cannot figure out when it happens, or after which sequence of clicks.

If you have any additional information on this bug, please let me know.
